crates/sui-indexer/src/handlers/committer.rs

```diff
@@ -212,6 +228,22 @@ async fn commit_checkpoints<S>(
         })
         .expect("Persisting data into DB should not fail.");
 
+    state
+        .update_watermarks_upper_bound(
+            PrunableTable::iter().collect(),
```

Comment:
what about tables that are un-prunable while we still want to track the `hi` watermark, like `epochs`?

Reply:
i think readers can just use the available data itself as the range, if needed

Reply:
it's counter-intuitive that watermarks is actually `prunable_table_watermarks` -- I propose to extend that to all tables if it's not the plan.

Reply:
we can do that in a separate pr

Reply:
i see what you mean
i dont think it'll hurt for pruner to prune a subset of the tables that committer pushes upper bounds to. For those unpruned tables, the `reader_lo` will remain at 0. Or perhaps yeah we can have a `prunable_table_watermarks` instead of a flat `watermarks`

Reply:
yeah I think having reader_lo at 0 seems benign too, and `watermarks` can be more useful covering all tables, for example if we want to check if all tables have reached certain checkpoint (except snapshot table), we can go to watermarks and run one quick query to do that.

Reply:
not trying to block you, and I can do the extension too if that helps

Reply:
oh no worries, better to hash it out now when the table doesnt exist yet
i like the idea of being able to check overall instance status